packer/install-dependencies.sh

```diff
@@ -401,10 +401,12 @@ K8S_VERSIONS="
 1.15.4-azs
 1.15.3
 1.15.3-azs
+1.14.8
 1.14.7
 1.14.7-azs
 1.14.6
```

Comment:
We need to remove both 1.14.6 versions. Until Azure Stack has a hyperkube image they are effectively unable to accept the resultant VHD.